ccmlib/node.py

```diff
@@ -556,6 +559,40 @@ def watch_log_for(self, exprs, from_mark=None, timeout=600, process=None, verbos
                         if process.returncode == 0:
                             return None
 
+    def watch_log_for_no_errors(self, exprs, from_mark=None, timeout=600, process=None, verbose=False, filename='system.log'):
+        """
+        Watch the log until one or more (regular) expression are found.
+        This methods when all the expressions have been found or the method
```

Comment:
There is something wrong with this sentence :-) Another verb?

Reply:
ill rewrite and fix watch_log_for

Reply:
Thanks :-)